### ISSUE TITLE
Make it possible to have hierachies in the PxWebApi

### ISSUE THREAD

**Issue by AsaArrhen** [PxTools/PxWebApi#107]
We would like to show hierarchies in the valuelists in PxWeb 2.0 but it is not supported in PxWebAPI2.0

**Comment by Maren-SSB** [comment]
We already have https://github.com/PxTools/pxtools-project/issues/161, so I don't know if we need this issue also?

**Comment by likp** [comment]
> We already have [PxTools/pxtools-project#161](https://github.com/PxTools/pxtools-project/issues/161), so I don't know if we need this issue also?

They are related. This issue is explicitly for the API which must provide the information before the change in PxWeb, so for me it is ok to keep this.

**Comment by Maren-SSB** [comment]
I agree and understand that the API and user interface are separate, but they also influence each other. API requirements will be influenced by the needs of the UI and the underlying user requirements and vice versa :-)

We also have other issues in the backlog that require API changes, without having their own API issues. But, we just have to agree on how we prefer to have it 👍
